app/scripts/modules/app/webpack.config.js

```diff
@@ -162,10 +162,6 @@ function configure(env, webpackOpts) {
           test: require.resolve('jquery'),
           use: [{ loader: 'expose-loader?$' }, { loader: 'expose-loader?jQuery' }],
         },
-        {
-          test: /ui-sortable/,
-          use: ['imports-loader?$UI=jquery-ui/ui/widgets/sortable'],
-        },
```

Comment:
did you check that this angularjs drag-sort feature still works?  I think it's only used in one or two places anymore.

Reply:
Yep, it is used in re-ordering links in config page and re-ordering deploy stages in pipeline config. I confirmed that both are working.